### DISCUSSION TITLE
大模型中chatglm3 与 zhipuai 有什么区别

### DISCUSSION THREAD

**Discussion by beidou987** [binary-husky/gpt_academic#1558] [Q&A]
chatglm3 是需要在线本地数据包吗？ zhipuai是指的智谱清言吗？我在https://open.bigmodel.cn/ 智谱AI开发平台申请的API应该选择哪个模型？感谢

**Comment by Nextstrain9000** [comment]
chatglm3是本地的，直接在config文件中填入"glm-4"就是在智谱官网调用api的
<img width="841" alt="搜狗截图20240219193916" src="https://github.com/binary-husky/gpt_academic/assets/140483081/beffe6ca-f700-4eaa-92b3-b24df9762c07">
